sources/subs/Mail.subs.php

```diff
@@ -141,7 +141,8 @@ function sendmail($to, $subject, $message, $from = null, $message_id = null, $se
 	}
 
 	// Return path, date, mailer
-	$headers .= 'Return-Path: ' . (!empty($modSettings['maillist_sitename_address']) ? $modSettings['maillist_sitename_address'] : (empty($modSettings['maillist_mail_from']) ? $webmaster_email : $modSettings['maillist_mail_from'])) . $line_break;
+	$return_path = (!empty($modSettings['maillist_sitename_address']) ? $modSettings['maillist_sitename_address'] : (empty($modSettings['maillist_mail_from']) ? $webmaster_email : $modSettings['maillist_mail_from'])); //We'll need this later
```

Comment:
Just to nitpicking: can we haz the comment on the line before the statement and not at the end of it? :angel: :eyes: 

I know I'm a pain in the arse sometimes... sorry. :blush: